app/assets/stylesheets/home.css.scss

```diff
@@ -44,7 +59,7 @@
 }
 
 .jumbotron {
-	margin: 0; 
+	margin: 0;
```

Comment:
Line should be indented with spaces, not tabs<br>Properties should be ordered background-color, margin, padding